packages/benchmarks/src/implementations/inline-styles/View.js

```diff
@@ -2,11 +2,7 @@
 import React from 'react';
 
 const compose = (s1, s2) => {
-  if (s1 && s2) {
-    return { ...s1, ...s2 };
-  } else {
-    return s1 || s2;
-  }
+  return s1 && s2 ? { ...s1, ...s2 } : s1 || s2;
```

Comment:
What is your opinion about adding parenthesis to the variables that are being checked?
for example:
`(s1 && s2) ? { ...s1, ...s2 } : (s1 || s2);`
I think it would be more clear, for the variables that are being tested :)